bindings/nodejs/tests/tests/OIC Server.js

```diff
@@ -22,10 +22,7 @@ var testUtils = require( "../assert-to-console" );
 var theResource;
 var anError = {};
 
-console.log( JSON.stringify( { assertionCount: 4 } ) );
-
-testUtils.assert( "strictEqual", soletta.sol_oic_server_init(), 0,
-	"sol_oic_server_init() is successful" );
+console.log( JSON.stringify( { assertionCount: 3 } ) );
```

Comment:
Sorry about that. I'll be sure to check Node.js bindings when I change any API in oic again.
Thanks
+1 for commti